<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextEdit" name="desc"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hl">
     <item>
      <layout class="QFormLayout" name="fl2">
       <item row="0" column="0">
        <widget class="QCheckBox" name="pick_1">
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="pick_2">
         <property name="text">
          <string/>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="pick_3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="pick_4">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="opt_1"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="opt_2"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="opt_3"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="opt_4"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="fl1">
       <item row="0" column="0">
        <widget class="QCheckBox" name="pick_5">
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="opt_5"/>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="pick_6">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="opt_6"/>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="pick_7">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="opt_7"/>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="pick_8">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="opt_8"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
